source/scss/components/_ibe-icons.scss

```diff
@@ -0,0 +1,15 @@
+.ibe-type-icons {
+  align-items: center;
+  display: inline-flex;
+  line-height: 26px;
+  vertical-align: middle;
+
+  i {
+    font-size: 26px;
+  }
+
+  > span {
+    font-size: 16px;
+    margin: 0 4px;
```

Comment:
Question: I don't understand the 4px. For what is it used?

Reply:
For spacing between the icons, to comply with the [Invision file](https://projects.invisionapp.com/d/main#/console/14456592/375083455/inspect) provided.

Reply:
I replaced it by a calculated value.